interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -151,50 +151,65 @@
-<input type="hidden" name="jumpdate" id="jumpdate" value="">
-<input type="hidden" name="viewtype" id="viewtype" value="[-php-]echo $viewtype;[-/php-]">
-[-php-]
-echo "   <a href='#' title='" . xl("New Appointment") . "' onclick='newEvt(1, 9, 00, $Date, 0, 0)' class='css_button'>
-            <i class='fa fa-plus' aria-hidden='true'></i></a>\n";
-echo "   <a href='#' title='" . xl("Search Appointment") . "'
-            onclick='top.restoreSession();location=\"index.php?module=PostCalendar&func=search\"' class='css_button'>
-            <i class='fa fa-search' aria-hidden='true'></i></a>\n";
-[-/php-]
-</div>
-
-
-<div id="dateNAV"">
-[-php-]
-if ($Date <> date('Ymd')) {
-[-/php-]
-<a href='#' name='bnsubmit' value='[-php-]echo xl ("Today")[-/php-]' onClick='GoToToday(theform);'  class='css_button'/><span>[-php-] echo xl("Today")[-/php-]</span></a>
-[-php-]
-}
-echo "   <a id='prevday' href='$PREV_DAY_URL' onclick='top.restoreSession()' title='" . xl("Previous Day") . "'>
-            <i class='fa fa-arrow-left' aria-hidden='true'></i></a>\n";
-$atmp = array_keys($A_EVENTS);
-echo dateformat (strtotime($atmp[0]),true);
-echo "   <a id='nextday' href='$NEXT_DAY_URL' onclick='top.restoreSession()' title='" . xl("Next Day") . "'>
-            <i class='fa fa-arrow-right' aria-hidden='true'></i></a>\n";
-[-/php-]
-</div>
+[-php-]require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";[-/php-]
 
-<div id="viewPicker">
-[-php-]
-echo "   <a href='#' id='printview' title='" . xl("View Printable Version") . "' class='css_button'>
-            <i class='fa fa-print' aria-hidden='true'></i></a>\n";
-echo "   <a href='#' title='" . xl("Refresh") . "' onclick='javascript:refreshme()' class='css_button'>
-            <i class='fa fa-refresh' aria-hidden='true'></i></a>\n";
-echo "   <a href='#' type='button' id='dayview' title='" .htmlspecialchars( xl('Day View'), ENT_QUOTES). "' class='css_button'/><span>".htmlspecialchars( xl('Day'), ENT_QUOTES)."</span></a>\n";
-echo "   <a href='#' type='button' id='weekview' title='" .htmlspecialchars( xl('Week View'), ENT_QUOTES). "' class='css_button'/><span>".htmlspecialchars( xl('Week'), ENT_QUOTES)."</span></a>\n";
-echo "   <a href='#' type='button' id='monthview' title='" .htmlspecialchars( xl('Month View'), ENT_QUOTES). "' class='css_button'/><span>".htmlspecialchars( xl('Month'), ENT_QUOTES)."</span></a>\n";
-[-/php-]
+<form name='theform' id='theform' action='index.php?module=PostCalendar&func=view&tplview=default&pc_category=&pc_topic=' method='post' onsubmit='return top.restoreSession()'>
+<nav class="navbar navbar-default">
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-xs-4">
+            <ul class="nav navbar-nav">
+                <li><a href='#' title='[-php-]echo xl("New Appointment")[-/php-]' onclick='newEvt(1, 9, 00, {$Date}, 0, 0)'><i class="fa fa-plus"></i>&nbsp;&nbsp;[-php-]echo xl("New Appointment")[-/php-]</a></li>
```

Comment:
if you want to be a really cool kid, could do the following now(to avoid the pure php bypass tags and do html escaping):
<li><a href='#' title='{xl t='New Appointment'|escape:'html'}' onclick='newEvt(1, 9, 00, {$Date}, 0, 0)'><i class="fa fa-plus"></i>&nbsp;&nbsp;{xl t='New Appointment'|escape:'html'}</a></li>